### ISSUE TITLE
adding orders via api calls

### ISSUE THREAD

**Issue by vasiliyb** [crxgames/WooCommerceClient#2]
Folks,  is there a way to create orders via the api?  I cant seem to find a way, only see API routes for get calls.

Thanks

**Comment by crxgames** [comment]
You are correct. I still need to add these. It should not be a whole lot of work to add though.
